spec/resqued/config/exit_event_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "spec_helper"
+require "resqued/worker"
+require "resqued/config/after_exit"
+
+describe do
+  before { evaluator.apply(config) }
+
+  context "after_exit" do
+    # Run the after_exit block.
+    #
+    #    after_exit do |worker_summary|
+    #      puts "#{worker_summary.alive_time_sec}"
+    #    end
+    #
+    # ignore calls to any other top-level method.
+
+    let(:config) { <<-END_CONFIG }
+      worker('one')
+      worker_pool(1)
+      queue('*')
+
+      after_exit do |worker_summary|
+        $after_exit_called = true
+        $worker_alive_time_sec = worker_summary.alive_time_sec
+        $exit_status = worker_summary.process_status.exitstatus
+      end
+    END_CONFIG
+
+    let(:evaluator) {
+      # In ruby versions < 3, Process::Status evaluates $? regardless of what status is returned by
+      # exitcode for a status created with Process::Status.allocate
+      fork do
+        0
```

Comment:
Can this `exit!`? The way it's written, the child process will start running from this point (including the rest of the enclosing block and maybe the rest of the test suite).

```suggestion
        exit!
```